WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -3567,6 +3567,13 @@ extension AztecPostViewController: AztecMoreCoordinatorDelegate {
     }
 }
 
+extension AztecPostViewController: StockPhotosPickerDelegate {
```

Comment:
I was thinking that, eventually, we might not need to actually expose this delegate. `AztecPostViewController` already implements `WPMediaPickerViewControllerDelegate`, we could eventually forward from   `StockPhotosPicker` to AztecPostViewController without exposing a new delegate, but that would be in another stage, and if it makes actual sense.

Reply:
That crossed my mind too at some point, sounds like a good possibility.